mapadroid/ocr/screenPath.py

```diff
@@ -403,6 +405,20 @@ def __handle_ptc_login(self) -> ScreenType:
         time.sleep(50)
         return ScreenType.PTC
 
+    def __handle_failure_screen(self, diff, global_dict) -> None:
```

Comment:
Returning None does not really match ScreenType ;)
`-> ScreenType:`

Reply:
Yeah, that happened during the "cleanup" :-)
Will be done in cleanup part 2.